docs/api-reference/v1.8/index.html

```diff
@@ -71712,21 +71480,21 @@ <h1 id="daemonset-v1beta2-apps">DaemonSet v1beta2 apps</h1>
 </thead>
 <tbody>
 <tr>
-<td>apps</td>
-<td>v1beta2</td>
+<td>extensions</td>
+<td>v1beta1</td>
 <td>DaemonSet</td>
```

Comment:
Isn't DaemonSet supposed to be apps, v1beta2?

Reply:
I think these are OK as is. (I think) this corrects the entries under OLD API VERSIONS. So the old API group version really should be extensions v1beta1.

Reply:
Ah, ok.

Reply:
I think it is: https://deploy-preview-5680--kubernetes-io-vnext-staging.netlify.com/docs/api-reference/v1.8/#daemonset-v1beta2-apps